frontend/src/app/main/ui/workspace/tokens/modals/themes.scss

```diff
@@ -212,19 +212,3 @@
   grid-template-columns: 0.6fr 1fr;
   gap: $s-12;
 }
-
-.label {
-  @include t.use-typography("body-small");
```

Comment:
Do we need to import typography anymore?

Reply:
Good catch!